mayastor/src/bdev/nexus/nexus_fn_table.rs

```diff
@@ -61,11 +61,11 @@ impl NexusFnTable {
         let nexus = unsafe { Nexus::from_raw(ctx) };
         match io_type {
             // we always assume the device supports read/write commands
-            io_type::READ | io_type::WRITE => true,
+            // allow NVMe Admin as it is needed for local replicas
+            io_type::READ | io_type::WRITE | io_type::NVME_ADMIN => true,
```

Comment:
do we need a local replica test below? This way it's opening it up for anything, eg what if a child does not support it (eg iscsi) ?

Reply:
When it reaches nexus_bdev::Nexus::nvme_admin, the call to spdk_bdev_nvme_admin_passthru should fail and hence fail the snapshot operation. It's not ideal but we don't support, or use, iSCSI children in k8s anyway.